### PR TITLE
Add Alchemy AA and zkEVM references

### PR DESCRIPTION
**Pull request by rthomare**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,6 +312,7 @@ Besides providing easy-to-integrate and scalable blockchain APIs, Infura empower
 1. GetBlock - [https://getblock.io/nodes/algo/](https://getblock.io/nodes/algo/)
 2. Pocket Network - [https://mainnet.portal.pokt.network](https://mainnet.portal.pokt.network)
 3. NOWNodes.io - [https://nownodes.io/](https://nownodes.io/)
+4. Alchemy - [https://alchemy.com](https://docs.alchemy.com/reference/polygon-zkevm-api-quickstart)
```
Comment (easeev) [inline README.md:315]: Ser I think you added it under `Algorand` not `Polygon zkEVM`

Comment (rthomare) [reply README.md:315]: Ahh 🤦

File: README.md [hunk 2]
```diff
@@ -104,6 +104,7 @@ Besides providing easy-to-integrate and scalable blockchain APIs, Infura empower
 1. Stackup.sh - [https://stackup.sh](https://stackup.sh)
 2. Etherspot - [https://etherspot.io](https://etherspot.io)
 3. Pimlico - [https://pimlico.io](https://pimlico.io)
+4. Alchemy - [https://www.alchemy.com](https://www.alchemy.com/account-kit)
```
Comment (arddluma) [inline README.md:107]: AFAIK AccountKit is a SDK which we do not list here :)

**Review by arddluma — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (arddluma) [reply README.md:107]: Do you mind actually adding Alchemy AA bundler service instead?

**Pull request closed without merging**